<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HologramInteractiveClass</class>
 <widget class="QMainWindow" name="HologramInteractiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HologramInteractive</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Model"/>
   </widget>
   <widget class="QMenu" name="menuHologram">
    <property name="title">
     <string>Hologram</string>
    </property>
    <addaction name="actionGeneration"/>
    <addaction name="actionReconstruction"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuHologram"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Model"/>
   <addaction name="actionGeneration"/>
   <addaction name="actionReconstruction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGeneration">
   <property name="icon">
    <iconset resource="holograminteractive.qrc">
     <normaloff>:/HologramInteractive/Resources/hologram_generation.png</normaloff>:/HologramInteractive/Resources/hologram_generation.png</iconset>
   </property>
   <property name="text">
    <string>Generation</string>
   </property>
   <property name="iconText">
    <string>Generation CGH</string>
   </property>
   <property name="toolTip">
    <string>Generation CGH</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionReconstruction">
   <property name="icon">
    <iconset resource="holograminteractive.qrc">
     <normaloff>:/HologramInteractive/Resources/hologram_reconstruction.png</normaloff>:/HologramInteractive/Resources/hologram_reconstruction.png</iconset>
   </property>
   <property name="text">
    <string>Reconstruction</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionOpen_Model">
   <property name="icon">
    <iconset resource="holograminteractive.qrc">
     <normaloff>:/HologramInteractive/Resources/open.ico</normaloff>:/HologramInteractive/Resources/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open Model</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="holograminteractive.qrc"/>
 </resources>
 <connections/>
</ui>
